### PR TITLE
chore: Fix broken test on Windows

### PR DESCRIPTION
**Pull request by dsherret**
`cat` does not exist by default on Windows. GitHub actions adds a lot of stuff to the Windows runner to make it work, which is why the CI is ok (I'm often surprised at what works in there).

### CODE REVIEW

File: test_util/src/lib.rs [hunk 1]
```diff
@@ -76,7 +76,10 @@ lazy_static! {
 }
 
 pub fn root_path() -> PathBuf {
-  PathBuf::from(concat!(env!("CARGO_MANIFEST_DIR"), "/.."))
+  PathBuf::from(concat!(env!("CARGO_MANIFEST_DIR")))
+    .parent()
+    .unwrap()
+    .to_path_buf()
```
Comment (dsherret) [inline test_util/src/lib.rs:82]: This just makes the paths that get outputted whiel the tests are running appear resolved.

File: cli/tests/089_run_allow_list.ts [hunk 2]
```diff
@@ -7,7 +7,9 @@ try {
 }
 
 const proc = Deno.run({
-  cmd: ["cat", "089_run_allow_list.ts"],
+  cmd: Deno.build.os === "windows"
+    ? ["cmd", "/c", "type", "089_run_allow_list.ts"]
+    : ["cat", "089_run_allow_list.ts"],
```
Comment (dsherret) [inline cli/tests/089_run_allow_list.ts:12]: I don't like doing these conditionals. Let me know if you can think of something that exists on all three operating systems by default.

Comment (lucacasonato) [reply cli/tests/089_run_allow_list.ts:12]: `curl` exists on all three now AFAIK. Or maybe `echo`?

Comment (dsherret) [reply cli/tests/089_run_allow_list.ts:12]: Oh, `echo` is good. Thanks!

**Review by lucacasonato — APPROVED** [review]

**Pull request merged**